src/arrayMethodSort.js

```diff
@@ -3,9 +3,41 @@
 /**
  * Implement method Sort
  */
+function compareAsStrings(first, second) {
+  const a = String(first);
+  const b = String(second);
+
+  if (a > b) {
+    return 1;
+  }
+
+  if (a < b) {
+    return -1;
+  }
+
+  return 0;
+}
+
 function applyCustomSort() {
-  [].__proto__.sort2 = function(compareFunction) {
-    // write code here
+  [].__proto__.sort2 = function(compareFunction = compareAsStrings) {
+    let counter;
+
+    do {
+      counter = 0;
```

Comment:
I suggest to use boolean value such a isContinue or smth like that to raise readability of the code

Reply:
Hi Volodymyr,
i am not really sure what you mean here ? Can you tip me on it ?